### DISCUSSION TITLE
Idea Posts: Probably not replacing the Suggestions on the Issue Tracker

### DISCUSSION THREAD

**Discussion by LlmDl** [TownyAdvanced/Towny#4553] [General]
So while this is still early days and Discussions might change, there's currently no Labels for Discussions. And, while you can convert an Issue ticket to a Discussion you cannot convert a Discussion to an Issue ticket. Therefore, if you have a fully-formed idea you want to have considered for adoption into Towny: make a Issue ticket using the suggestion option. If your idea is not fully-formed and you want to see what the community has to say, make an Idea thread in the Discussions.

**Comment by TheBeeCollector** [comment]
> So while this is still early days and Discussions might change, there's currently no Labels for Discussions. And, while you can convert an Issue ticket to a Discussion you cannot convert a Discussion to an Issue ticket. Therefore, if you have a fully-formed idea you want to have considered for adoption into Towny: make a Issue ticket using the suggestion option. If your idea is not fully-formed and you want to see what the community has to say, make an Idea thread in the Discussions.

The github team is currently looking into implementing shared labels between discussions, PR's and issues:
https://github.com/github/feedback/discussions/19#discussioncomment-128146

**Reply by LlmDl** [reply]
That's nice, I've also discovered you can quickly create an issue ticket that references a discussion post. So if/when some of you folks discuss a concept idea into something fully-formed it will be fairly easy to make a ticket out of it.
